### DISCUSSION TITLE
Interface Difficulty

### DISCUSSION THREAD

**Discussion by AdityaSrivastavDS** [microsoft/SATechnicalOnboarding#67] [Questions]
I am facing difficulty in how to check my milestones and my growth.
And did we recieve swag packs automatically or we have to request for it?

**Comment by AnthonyByansi** [comment]
Hi @AdityaSrivastavDS,

Your milestones can be found on our ambassador website within your profile. Upon reaching the Alpha milestone, you'll gain access to $150 in monthly Azure credits and LinkedIn learning resources. Upon reaching the Beta Milestone, we will automatically ship swag to the address you've provided in your profile. Please make sure the address is accurate, and there's no need to request the swag; it will be shipped to you directly.

**Comment by NanciRamirez** [comment]
hola, estoy intentando crear mi cuenta en azure, pero no me deja continuar pues me indica que el ID de mi universidad no se encuentra q puedo hacer en este caso?

**Comment by NanciRamirez** [comment]
Hello, I'm trying to create my account in Azure, but it won't let me continue because it tells me that my university ID is not found what can I do in this case?

**Reply by AnthonyByansi** [reply]
Hi @NanciRamirez! Check out https://github.com/microsoft/SATechnicalOnboarding/discussions/20#discussioncomment-7442451

**Reply by AdityaSrivastavDS** [reply]
Check if your university is a part of the Azure for Education program: Azure offers special benefits for educational institutions through the Azure for Education program. Make sure that your university is part of this program and that you're using the correct method to sign up.

**Reply by NanciRamirez** [reply]
thanks, it's ready
